scripts/plugins/settings.py

```diff
@@ -20,16 +20,14 @@
     if BITOPS_CL_args.bitops_config_file is not None
     else "bitops.config.yaml"
 )
-with open(BITOPS_config_file, "r") as stream:
+with open(BITOPS_config_file, "r", encoding="utf8") as stream:
```

Comment:
Should this be made into a BitOps config value?

Reply:
Nope, the main point is that utf8 should be set everywhere as a default in python. If not specified, the default system encoding is used. Users shouldn't configure it, it's more of the systems setting better to be hardcoded for their safety.

See pylint `W1514` explanation: https://pylint.pycqa.org/en/latest/user_guide/messages/warning/unspecified-encoding.html

Reply:
Neat 👍 Thanks for the info